package.json

```diff
@@ -12,7 +12,7 @@
   },
   "license": "MIT",
   "private": true,
-  "packageManager": "pnpm@9.15.1",
+  "packageManager": "pnpm@9.15.3",
```

Comment:
should we lock the engines version?

Reply:
Engines spec violations lead to process exit which can be pretty aggressive. The `packageManager` field already forces user's machine to dynamically use the specified package manager and version (assuming `corepack` is enabled).

We could discuss engines in another PR maybe, but don't think it has too much impact. I'm not certain what the point of pnpm engine spec with `packageManager` spec is. I guess if developer or CI is not using `corepack` properly then the engines spec is a kind of fallback/sanity check. Seems like a flakey reason though.

Reply:
I've noticed this one after pushing a change to `engines`. I think we also need `engines` here in the monorepo root because it affects local development (only). We had before cases where developers had older versions locally, and then it led to weird lock file diffs.